server/helpers/http.js

```diff
@@ -16,3 +17,22 @@ export const httpResponse = (res, options) => {
   res.end();
   return response;
 };
+
+/**
+ * @description Returns an server error
+ *
+ * @param {object} res HTTP response object
+ * @param {object} error error object
+ * @returns {object} HTTP response
+ * @method serverError
+ */
+export const serverError = (res, error) => {
+  console.log(error.message);
+  return res.status(error.status || 500).json({
+    success: false,
+    message:
+      error && cleanString(process.env.NODE_ENV) === "production"
```

Comment:
This line is not very readable. I have told @mrkingz  to remove this function and you were right beside me when I was doing it.

better 
`const isProudction = process.env.NODE_ENV === "production"`

now we can do 
```
message:
      error && isProduction
```